doc/api/esm.md

```diff
@@ -92,6 +92,9 @@ if the nearest parent `package.json` contains `"type": "module"`.
 ```js
 // my-app.js, part of the same example as above
 import './startup.js'; // Loaded as ES module because of package.json
+
+// Extensionless. Requires 'node --es-module-specifier-resolution=node'
+import './startup'; // Loaded as ES module because of package.json
```

Comment:
Extensionless files, as documented above, are different from extensionless specifiers (which you are documenting here). Extensionless files mean the file path itself has no extension e.g. `/path/to/file`, not the specifier e.g. `'./file'` where path is `/path/to/file.js`.

Reply:
Ah I see. Yeah it could definitely be clarified.

Reply:
Done. Had trouble writing it succinctly so let me know if it makes sense.